scripts/install-ilab-with-cuda.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/env bash
+
+# Before calling this script, we must set an alias for `python` like so:
+# alias python='python3.11'
+
+# Validate that the CUDA libs + drivers can be found after configuring CUDA paths
+export CUDA_HOME="/usr/local/cuda"
+export LD_LIBRARY_PATH="$LD_LIBRARY_PATH:$CUDA_HOME/lib64:$CUDA_HOME/extras/CUPTI/lib64"
+export PATH="$PATH:$CUDA_HOME/bin"
+nvidia-smi
```

Comment:
it won't "verify" anything unless we `set -e` for the script (which we should do to catch errors).